lib/hydra/works/models/concerns/generic_file/characterization/video.rb

```diff
@@ -0,0 +1,20 @@
+include RDF
```

Comment:
This is a pattern that I don't often see in Hydra code. I have a preference for striking all the "include RDF" lines in the characterization modules and then using the explicit `RDF::Vocab::NFO` (for instance) below and in the other files where this pattern is used.